sdk/src/test/java/com/horizen/storage/StorageTest.java

```diff
@@ -1,72 +1,159 @@
 package com.horizen.storage;
 
-import com.horizen.fixtures.IODBStoreFixtureClass;
+import com.horizen.fixtures.StoreFixtureClass;
+import com.horizen.storage.leveldb.VersionedLevelDbStorageAdapter;
 import com.horizen.utils.ByteArrayWrapper;
 
+import java.io.File;
 import java.util.*;
 import com.horizen.utils.Pair;
 
 import org.junit.*;
+
 import static org.junit.Assert.*;
 
 public class StorageTest {
 
-    IODBStoreFixtureClass storageFixture = new IODBStoreFixtureClass();
+    StoreFixtureClass storageFixture = new StoreFixtureClass();
 
     @Test
     public void testStorage() {
-        Storage s = new IODBStoreAdapter(storageFixture.getStore());
+        VersionedLevelDbStorageAdapter s = storageFixture.getStorage();
         assertTrue("Storage expected to be empty.", s.isEmpty());
 
-        ByteArrayWrapper version1 = storageFixture.getVersion();
-        List<Pair<ByteArrayWrapper,ByteArrayWrapper>> u1 = storageFixture.getKeyValueList(3);
+        assertFalse("Version of empty storage shall be none", s.lastVersionID().isPresent());
+
+        ByteArrayWrapper version0 = storageFixture.getVersion();
```

Comment:
I would prefer to add some comments before each logical section like:
// Test 1: add entries to empty storage
......
//  Test 2: add entries to non-empty storage
.....
// Test 3: update existing entries
and so on. Just to follow the test cases faster.

Reply:
done